__test__/members.test.ts

```diff
@@ -0,0 +1,346 @@
+        const membersDataValuesMock: any[][] = getRandomlyGeneratedMemberTable(2, 1);
+        const membersDataRangeMock: Range = getRandomlyGeneratedRange();
+        membersDataRangeMock.getValues = jest.fn(() => membersDataValuesMock);
+        const membersTableSheetMock: Sheet = getRandomlyGeneratedSheet();
+        membersTableSheetMock.getName = jest.fn(() => "Members");
+        membersTableSheetMock.getDataRange = jest.fn(() => membersDataRangeMock);
+
+        membersDataValuesMock[1][NAME_COLUMN_INDEX] = "name1";
+        membersDataValuesMock[1][ALTERNATE_NAMES_COLUMN_INDEX] = "alias1,alias2";
+        membersDataValuesMock[2][NAME_COLUMN_INDEX] = "name2";
+        membersDataValuesMock[2][ALTERNATE_NAMES_COLUMN_INDEX] = "alias3,alias4";
+
+        const couplesDataValuesMock: any[][] = getRandomlyGeneratedAliasTable(0);
+        const couplesDataRangeMock: Range = getRandomlyGeneratedRange();
+        couplesDataRangeMock.getValues = jest.fn(() => couplesDataValuesMock);
+        const couplesTableSheetMock: Sheet = getRandomlyGeneratedSheet();
+        couplesTableSheetMock.getName = jest.fn(() => "Couples");
+        couplesTableSheetMock.getDataRange = jest.fn(() => couplesDataRangeMock);
+
+        const sheetsMock: Sheet[] = Array.from({length: 10}, getRandomlyGeneratedSheet);
+        sheetsMock.push(membersTableSheetMock);
+        sheetsMock.push(couplesTableSheetMock);
+        jest.mock("../src/main/scan", () => ({
+            getAllSpreadsheetTabs: jest.fn(() => sheetsMock),
+        }));
+
+        const setScriptPropertyMock: Mock = jest.fn();
+        jest.mock("../src/main/propertiesService", () => ({
+            getScriptProperty: jest.fn(),
+            setScriptProperty: setScriptPropertyMock,
+        }));
+
+        const { loadMembersFromOnestopIntoScriptProperties } = require('../src/main/members');
+        loadMembersFromOnestopIntoScriptProperties();
+
+        const expected: AliasMap = {
+            alias1: ["name1"],
+            alias2: ["name1"],
+            alias3: ["name2"],
+            alias4: ["name2"],
+        };
+        expect(setScriptPropertyMock).toHaveBeenCalledWith("ALIASES_MAP", JSON.stringify(expected));
+    });
+
+    it("should map an alternate name to multiple people when multiple people share the same alternate name", () => {
+        const membersDataValuesMock: any[][] = getRandomlyGeneratedMemberTable(2, 1);
+        const membersDataRangeMock: Range = getRandomlyGeneratedRange();
+        membersDataRangeMock.getValues = jest.fn(() => membersDataValuesMock);
+        const membersTableSheetMock: Sheet = getRandomlyGeneratedSheet();
+        membersTableSheetMock.getName = jest.fn(() => "Members");
+        membersTableSheetMock.getDataRange = jest.fn(() => membersDataRangeMock);
+
+        membersDataValuesMock[1][NAME_COLUMN_INDEX] = "name1";
+        membersDataValuesMock[1][ALTERNATE_NAMES_COLUMN_INDEX] = "alias1,alias2";
```

Comment:
for tests like these, sometimes it helps to use more "real" looking values to make it easier to read, e.g. "Kevin L., Kevin", "Kevin H, Kevin". a little bit faster to read and tell where the duplication is

Reply:
Names and aliases used in the tests replaced with more realistic names generated by chatgpt. Didn't want to use our actual names for privacy sake